components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/SAMLSSOService.java

```diff
@@ -238,6 +239,7 @@ public void doSingleLogout(String sessionId, String issuer) throws IdentityExcep
         //send logout requests to all session participants
         LogoutRequestSender.getInstance().sendLogoutRequests(singleLogoutReqDTOs.toArray(
                 new SingleLogoutRequestDTO[singleLogoutReqDTOs.size()]));
+        // TODO : Check for back-channel logout and filter the session removal.
```

Comment:
why TODO added?

Reply:
As this PR was sent to a feature branch this TODO was added. This is addressed later in https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/208.